src/js/controller/model.js

```diff
@@ -101,17 +106,12 @@ const Model = function() {
                 var itemMeta = this.get('itemMeta');
                 Object.assign(itemMeta, data.metadata);
                 break;
-            case MEDIA_BUFFER_FULL:
-                // media controller
-                mediaModel.change('playAttempt', (mediaModelChanged, playAttempt) => {
-                    if (playAttempt) {
-                        mediaModelChanged.off('playAttempt', null, this);
-                        this.playVideo();
-                    }
-                }, this);
-                this.setPlaybackRate(this.get('defaultPlaybackRate'));
-                break;
             case MEDIA_TIME:
+                // Don't sent time event on Android before real duration is known
```

Comment:
What's the side effect that this causes on Android? Seems like it'd be beneficial across the board.

Reply:
We should show the correct duration (and Live mode) as soon as possible. Android incorrectly shows live mode for HLS streams until it finishes manifest parsing. Other browsers would show duration of 0 instead of live mode before transitioning.